boteval/templates/admin/mturk/_macros.html

```diff
@@ -1,4 +1,4 @@
-{% macro render_assignment(asgn, qtypes=None, crowd_name='mturk') %}
+{% macro render_assignment(asgn, qtypes=None, base_pay = 0.0, pay_per_hour = 0.0, bonus_pay='', crowd_name='mturk') %}
```

Comment:
these parameters are overridden in mturk/HIT.html, so it won't matter in practice, but I think it would be good to not have default values and instead raise errors if these parameters are not provided through the config.